Extend Azure Monitor scaler to support custom metrics (#468)
kedacore/keda-docs: content/docs/2.4/scalers/azure-monitor.md

- `tenantId` - Id of the tenant that contains the Azure resource. This is used for authentication.
- `subscriptionId` - Id of Azure subscription that contains the Azure resource. This is used for determining the full resource URI.
- `resourceGroupName` - Name of the resource group for the Azure resource.

> - `metricName` - Name of the metric. Can either be an officially supported metric, as can be found in the [official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported), or a custom metric name when querying Azure Monitor [Custom metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-custom-overview).

Reviewer: ```suggestion
- `metricName` - Name of the metric to query.
  - Azure metrics are available as a list in the [official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported)
  - Custom metric name when querying Azure Monitor [Custom metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-custom-overview).
```
